setup.py

```diff
@@ -483,40 +550,52 @@ def main():
                 os.path.join("src", "algorithms", "banditpam.cpp"),
                 os.path.join("src", "algorithms", "banditpam_orig.cpp"),
                 os.path.join("src", "algorithms", "fastpam1.cpp"),
-                os.path.join("src", "python_bindings",
-                             "kmedoids_pywrapper.cpp"),
+                os.path.join(
+                    "src", "python_bindings", "kmedoids_pywrapper.cpp"
+                ),
                 os.path.join("src", "python_bindings", "medoids_python.cpp"),
-                os.path.join("src", "python_bindings",
-                             "build_medoids_python.cpp"),
+                os.path.join(
+                    "src", "python_bindings", "build_medoids_python.cpp"
+                ),
                 os.path.join("src", "python_bindings", "fit_python.cpp"),
                 os.path.join("src", "python_bindings", "labels_python.cpp"),
                 os.path.join("src", "python_bindings", "steps_python.cpp"),
                 os.path.join("src", "python_bindings", "loss_python.cpp"),
                 os.path.join("src", "python_bindings", "cache_python.cpp"),
-                os.path.join("src", "python_bindings",
-                             "swap_times_python.cpp"),
+                os.path.join(
+                    "src", "python_bindings", "swap_times_python.cpp"
+                ),
             ],
             include_dirs=include_dirs,
-            library_dirs=[
-                os.path.join("/", "usr", "local", "lib"),
-                # For Mac Github runners that install locally
-                # (not build wheels) - testing for macos-10.15
-                os.path.join("/", "usr", "local", "Cellar", "libomp",
-                             "15.0.2", "lib"),
-
-                # for macos-latest
-                os.path.join("/", "usr", "local", "Cellar", "libomp",
-                             "15.0.7", "lib"),
-            ],
+            library_dirs=library_dirs,
             libraries=libraries,
             language="c++1z",  # TODO: modify this based on cpp_flag(compiler)
-            extra_compile_args=["-static-libstdc++"],
+            extra_compile_args=cpp_args,
+            extra_link_args=[],  # Wrong pass (BuildExt sets extra_link_args)
         )
     ]
 
     with open(os.path.join("docs", "long_desc.rst"), encoding="utf-8") as f:
         long_description = f.read()
 
+    my_data_files = None
+    if sys.platform == "win32":
+        my_data_files = [
+            ("docs", [os.path.join("docs", "long_desc.rst")]),
+            (
+                "",
+                [
+                    os.path.join(
+                        os.getcwd(),
+                        r"headers\armadillo\examples\lib_win64"
+                        + r"\libopenblas.dll",
+                    )
+                ],
+            ),
+        ]
+    else:
+        my_data_files = [("docs", [os.path.join("docs", "long_desc.rst")])]
+
```

Comment:
The first data file is common to all platforms, so you can initialize `my_data_files` to that and `append` if on `win32`